README.md

```diff
@@ -37,19 +37,74 @@ git clone https://github.com/PetJournal/petjournal.api.git
 docker compose -f docker-compose.dev.yml up -d
 ```
 
-4. Garanta que o [**Node.js**](https://nodejs.org/en/download/) está instalado em sua máquina e então habilite o gerenciador de pacotes yarn usando o comando `corepack enable`;
+4. Garanta que o [**Node.js**](https://nodejs.org/en/download/) está instalado em sua máquina e então habilite o gerenciador de pacotes `yarn` usando o comando `corepack enable`;
 
 5. Acesse a pasta do projeto com seu terminal;
 
 6. Rode o comando `yarn` para instalar as dependências do projeto;
 
-7. Após a correta configuração acima, rode a aplicação com o comando `yarn dev`. Você deve receber a seguinte mensagem de confirmação:
+7. Faça as migrações no banco de dados;
+```bash
+npx prisma migrate dev
```

Comment:
change to `yarn prisma migrate dev` since we are using yarn in the beginning

Reply:
Maybe, `yarn migrate` be a good alias to abstract this command.